server.js

```diff
@@ -0,0 +1,18 @@
+// const express = require('express');
+// const path = require('path');
+
+// const app = express();
+
+// if (process.env.NODE_ENV !== 'production') {
+//   const webpackMiddleware = require('webpack-dev-middleware');
+//   const webpack = require('webpack');
+//   const webpackConfig = require('./webpack.config.js');
+//   app.use(webpackMiddleware(webpack(webpackConfig)));
+// } else {
+//   app.use(express.static('dist'));
+//   app.get('*', (req, res) => {
+//     res.sendFile(path.join(__dirname, 'dist/index.html'));
+//   });
+// }
+
+// app.listen(process.env.PORT || 3050, () => console.log('Listening'));
```

Comment:
It’s better to delete commented code. If you need to review a previous version of your code, use git history
See this [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from Pathwright.

Since this file is not in use, please remove it. 🙂